assets/src/edit-story/migration/migrations/v0020_backgroundColorToPage.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function backgroundColorToPage({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage(page) {
+  const { elements, defaultBackgroundElement } = page;
+  const defaultBackground = {
+    type: 'solid',
+    color: { r: 0, g: 0, b: 0, a: 0 },
+  };
+  const backgroundColor = elements[0]?.isDefaultBackground
+    ? elements[0].backgroundColor
+    : defaultBackgroundElement?.backgroundColor;
+  return {
+    ...page,
+    backgroundColor: backgroundColor ? backgroundColor : defaultBackground,
```

Comment:
Should we also add the default white color to [any newly created page in `createPage`](https://github.com/google/web-stories-wp/blob/939ff090298e1bdc69019e2005279c342fdc5417/assets/src/edit-story/elements/index.js#L64-L66) or is it safe to assume, that no bg means white?

And if so, then this default here should also just not do anything if there's no background element to copy from.

Reply:
Yes, we should add the default white color to any newly created page!